rust/processor/migrations/2023-07-19-190707_add_ans_is_primary/up.sql

```diff
@@ -0,0 +1,19 @@
+ALTER TABLE current_ans_lookup
+ADD COLUMN is_primary BOOLEAN NOT NULL DEFAULT FALSE,
+ADD COLUMN is_deleted BOOLEAN NOT NULL DEFAULT FALSE;
+
+CREATE TABLE ans_lookup (
+    transaction_version BIGINT NOT NULL,
+    domain VARCHAR(64) NOT NULL,
+    -- if subdomain is null set to empty string
+    subdomain VARCHAR(64) NOT NULL,
+    registered_address VARCHAR(66),
+    expiration_timestamp TIMESTAMP,
+    token_name VARCHAR(128) NOT NULL,
+    is_primary BOOLEAN,
+    is_deleted BOOLEAN NOT NULL DEFAULT FALSE,
+    transaction_timestamp TIMESTAMP NOT NULL,
+    inserted_at TIMESTAMP NOT NULL DEFAULT NOW(),
+    -- Constraints 
+    PRIMARY KEY (transaction_version, domain, subdomain)
```

Comment:
Why not just use transaction_version and write_set_change_index to conform to the pattern? It'll also be easier to find.

Reply:
Because an ANS creation can exist in 2 wsc indices (the name record and the primary key reverse lookup). We could also split them up into 2 rows.